lua/nvim-tree/utils.lua

```diff
@@ -354,6 +354,27 @@ function M.focus_file(path)
   require("nvim-tree.view").set_cursor { i + 1, 1 }
 end
 
+function M.focus_node_or_parent(node)
```

Comment:
Please add doc annotations. See `is_nvim_tree_buf` for an example.

Reply:
Sure, I didn't do it because most of the other functions don't have them. Will all of them eventually have annotations?

Reply:
Many thanks. Ideally we would like documentation everywhere but that will be a slow process...